rasa/core/training/interactive.py

```diff
@@ -329,6 +346,15 @@ def _selection_choices_from_intent_prediction(
     return await _ask_questions(question, sender_id, endpoint)
 
 
+async def _request_free_text_utterance(
+    sender_id: Text, endpoint: EndpointConfig, action: Text
+) -> Text:
+    question = questionary.text(
+        "Please type the message for your " "new action {}".format(action)
```

Comment:
hm something went weird with the formatting here -- Also should we say "for your new utter template" instead? or just template?